packages/gatsby-plugin-google-tagmanager/README.md

```diff
@@ -14,6 +14,10 @@ plugins: [
     resolve: `gatsby-plugin-google-tagmanager`,
     options: {
       id: 'YOUR_GOOGLE_TAGMANAGER_ID',
+      // want GTM to render on dev? (localhost:8000)
```

Comment:
A few other changes.

* Make comments full sentences
* spell out "development"
* make same change to "include" from render
* No need to include "localhost:8000"